src/bundle/settings/index.scss

```diff
@@ -25,6 +25,9 @@
     width: 300px;
 
     &__mode-help {
+        position: relative;
+        top: 8px;
```

Comment:
why 8px added?

Reply:
Because help-mark was at the top of menu-item